src/stream/src/executor/source/source_executor.rs

```diff
@@ -99,7 +101,8 @@ impl<S: StateStore> SourceExecutor<S> {
         Ok(Self {
             ctx,
             source_id,
-            source_desc,
+            source_builder,
+            row_id_index: None,
```

Comment:
This looks not that "Rust". 🤣 How about removing this field and make it a parameter of `refill_row_id_column`?

Reply:
You are right : )